website/docusaurus.config.js

```diff
@@ -14,18 +14,13 @@ if (process?.env?.VERCEL_ENV === "preview" && process?.env?.VERCEL_BRANCH_URL) {
   SITE_URL = `http://localhost:3000`;
 }
 
-var GIT_BRANCH;
-if (!process.env.CONTEXT || process.env.CONTEXT == "production") {
-  GIT_BRANCH = "current";
-} else {
-  GIT_BRANCH = process.env.HEAD;
-}
+const GIT_BRANCH = process?.env?.VERCEL_GIT_COMMIT_REF;
 
 let { ALGOLIA_APP_ID, ALGOLIA_API_KEY, ALGOLIA_INDEX_NAME } = process.env;
 
 let metatags = [];
 // If Not Current Branch, do not index site
-if (GIT_BRANCH !== "current") {
+if (GIT_BRANCH && GIT_BRANCH !== "current") {
```

Comment:
Would be great to have a second set of eyes here. This should set the noindex metatag only if GIT_BRANCH is set, AND GIT_BRANCH is not === to `current`.

`VERCEL_GIT_COMMIT_REF` returns the branch name: https://vercel.com/docs/environment-variables/system-environment-variables#VERCEL_GIT_COMMIT_REF